dbfit-java/build.gradle

```diff
@@ -294,6 +308,83 @@ project('netezza') {
         compile project(':dbfit-java:core')
         runtime 'org.netezza:nzjdbc3:7.2@jar'
     }
+
+    task connectionPropertyFilename {
+        ext.propertiesFile = 'TestDbConnectionDbFitNetezza.properties'
+    }
+
+    task copyConnectionFile(dependsOn: connectionPropertyFilename) {
+        copy {
+            from connectionPropertyFilename.propertiesFile + '.default'
+            into '.'
+            rename '(.*).default', '$1'
+        }
+
+        copy {
+            from connectionPropertyFilename.propertiesFile + '.custom'
+            into '.'
+            rename '(.*).custom', '$1'
+        }
+    }
+
+    task connectionProperties(dependsOn: copyConnectionFile) {
+        doLast {
+            ext {
+                jdbcDriverClass = 'org.netezza.Driver'
+                jdbcUrlPrefix = 'jdbc:netezza://'
+                delimiterType = 'row'
+            }
+
+            def props = new Properties()
+            file(connectionPropertyFilename.propertiesFile).withInputStream { props.load(it) }
+            def propService = props.getProperty('service')
+            def propServiceParts = propService.tokenize(':')
+            def propServiceHost = propServiceParts[0]
+            def propServicePort = propServiceParts[1]
+            def propDatabase = props.getProperty('database')
+
+            ext {
```

Comment:
I guess not as the `def` statement don't result in an extra property being created. I was trying to be very explicit about which ones were creating properties.

I'll try merging them.